nailgun/entities.py

```diff
@@ -4332,6 +4332,25 @@ def bulk_available_incremental_updates(self, synchronous=True, timeout=None, **k
         response = client.post(self.path('bulk/available_incremental_updates'), **kwargs)
         return _handle_response(response, self._server_config, synchronous, timeout)
 
+    def play_roles(self, synchronous=True, timeout=None, **kwargs):
```

Comment:
Would suggest to rename this func be specific, and mind moving this func to the end of the class where all ansible_role related func exists for readability?
```suggestion
    def play_ansible_roles(self, synchronous=True, timeout=None, **kwargs):
```